pyproject.toml

```diff
@@ -3,13 +3,13 @@ requires = ["setuptools>=61", "wheel", "tomli;python_version<'3.11'", "setuptool
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "IMAS-ParaView"
+name = "imas-paraview"
 authors = [
     {name = "Panchumarti Jaswant EXT", email = "jaswant.panchumarti@iter.org"},
```

Comment:
The email link on https://test.pypi.org/project/imas-paraview/ now says "ITER Organization" with mailto:jaswant.panchumarti@iter.org.

Is that desired? Or do you prefer the `imas-support@iter.org` link?